packages/monaco/src/index.tsx

```diff
@@ -101,6 +102,22 @@ export function CodePanel(props: CodePanelProps) {
   };
   const hasFailingTest = tsErrors?.some((e) => e.length) ?? false;
 
+  useEffect(() => {
+    const onSubmit = (e: KeyboardEvent) => {
+      if ((e.ctrlKey || e.metaKey) && e.code === 'KeyY') {
+        e.preventDefault();
+        if (submitButtonRef.current) {
```

Comment:
you don’t need to simulate clicking the button. just call the same function

Reply:
I tried calling the handleSubmit funtion but when directly calling the handleSubmit the tsErros will be undefined  so every sumbit passed

Reply:
hmm i’ll take a look

Reply:
quick update it's now working the linter didn't give me any warning regarding missing dependency.